nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-flow-version.js

```diff
@@ -1843,6 +1843,15 @@
             }).promise();
         },
 
+        /**
+         * Downloads the latest versioned flow with the specified process group id
+         *
+         * @param processGroupId
+         */
+        downloadFlowVersion: function (processGroupId) {
```

Comment:
This does not appear to be invoked. I'd recommend removing it as it would need to be updated to support a One Time Password if we wanted to leave it in place.